lib/Basics/files.cpp

```diff
@@ -276,16 +276,25 @@ bool TRI_IsSymbolicLink(char const* path) {
 /// @brief creates a symbolic link
 ////////////////////////////////////////////////////////////////////////////////
 
-bool TRI_CreateSymbolicLink(std::string const& target, std::string const& linkpath, 
-                            std::string& error) {
+bool TRI_CreateSymbolicLink(std::string const& target,
+                            std::string const& linkpath, std::string& error) {
 #ifdef _WIN32
-  // TODO : check if a file is a symbolic link - without opening the file
-  return false;
+  bool created = ::CreateSymbolicLinkA(linkpath.c_str(), target.c_str(), 0x0);
```

Comment:
use `CreateSymbolicLinkW()` + utf8=>utf16 conversion as all the other functions do too.

Reply:
How will the std::string contain something else than ASCII or utf-8? Do we encode utf16 into std::basic_string<char> so we need to re-encode it to wchar_t when using the windows API?

Reply:
The ANSI Windows APIs (A suffix) expect strings to be encoded based the (locale dependent) active code page (see https://docs.microsoft.com/en-us/windows/desktop/intl/code-pages). Since you internally use utf-8 encoded strings you should convert them to utf-16 and use the Unicode APIs (W suffix).